scripts/linters/html_linter.py

```diff
@@ -358,6 +358,28 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.html_filepaths
 
+    @staticmethod
+    def _get_trimmed_error_messages(lint_messages):
+        """Remove extra bits from htmllint error messages.
+
+        Args:
+            lint_messages: str. Messages returned by the html linter.
+
+        Returns:
+            str. A string with the trimmed error messages.
+        """
+        error_messages = ''
+        # Extracting messages by removing number of files linted and number of
+        # error messages from the end of message. Becuase we last two lines
+        # like this:
+        # 1 error found.
+        # [htmllint] 1 files linted.
+        # Hence removing last two lines from the message.
+        messages = lint_messages.split('\n')[:-2]
+        for message in messages:
+            error_messages += message + '\n'
```

Comment:
See above -- have this as a list and return `'\n'.join(error_messages) + '\n'` or similar.

In general, plural names indicate lists.